ssl/t1_trce.c

```diff
@@ -1251,8 +1281,23 @@ static int ssl_print_ticket(BIO *bio, int indent,
     msg += 4;
     BIO_indent(bio, indent + 2, 80);
     BIO_printf(bio, "ticket_lifetime_hint=%u\n", tick_life);
+    if (SSL_IS_TLS13(s)) {
+        unsigned int ticket_age_add;
```

Comment:
Sorry it been doing things without a blank line for years and I backslide into my old ways occasionally...